examples/extract_data/src/Extract data login.md

```diff
@@ -0,0 +1,162 @@
+Extract data from a webpage requiring login
```

Comment:
This is what I'd expect for the fern docs and not the README.md. You should rename this file to README.md and have it be closer to what is done [here](https://github.com/airtop-ai/examples-python/blob/main/examples/simple-interactions/src/README.md). 

Having a short overview (1st paragraph) is still useful.